aeneas/src/mach/MachBackend.v3

```diff
@@ -91,8 +91,7 @@ class MachFrame(conv: MachCallConv, wordSize: int) {
 	def allocSpill(regClass: RegClass) -> int {
 		match (regClass) {
 			I64, F64 => {
-				var numSpill = 64 / wordSize;
-				if (numSpill == 0) numSpill = 1; // in case word size is larger than 64-bit XXX: is it required?
+				var numSpill = if(wordSize >= 64, 1, 2); // XXX: assumes at least 32-bit word size
```

Comment:
An alternative would be to pass an {Alignment} object. There's one in {AddressSpace.addrAlign}; then you can do {align.alignUp({4, 8}) >> align.shift).

Reply:
You mean pass it to `allocSpill()` or during creation as a parameter to `MachFrame`?

Reply:
Upon creation.

Reply:
Done. But slight aside is that I think we're assuming that `RegClass.REF` is always 32bits/4bytes as opposed to being architecture dependent.

Reply:
Yeah, I was going to suggest adding refSize as an argument too. It's in the MachProgram.

Reply:
I wonder if it's more elegant to pass the `AddressSpace` around instead. Everything we want is encapsulated by it (`addressSize` and `addrAlign`).

Reply:
Also do we care about annotating everything with `IS_64` on a 64-bit architecture? It doesn't really matter as the spill size is 64-bits so we don't have the special case anymore.

Reply:
The refSize is (or will be) independent of the address space size, as I plan to implement compressed references on 64-bit to save memory.

I think we can get rid of IS_64 in the future when we deprecate the OldCodeGen. In the new code gen, there is always a VReg available when generating moves, which has a register class. I think the main culprit for that mess was the MoveResolver, which is kind of folded into SimpleRegAlloc now.

Reply:
Cool. I've just pushed the small fix then. I think the PR should be good to go.